ocs_ci/ocs/ocp.py

```diff
@@ -465,12 +465,25 @@ def wait_for_resource(
                 1 in ContainerCreating) the function will continue to next
                 iteration to wait for only 2 resources in running state and no
                 other exists.
+            error_condition (str): State of the resource that is sampled
+                from 'oc get <kind> <resource_name>' command, which makes this
+                method to fail immediately without waiting for a timeout. This
+                is optional and makes sense only when there is a well defined
+                unrecoverable state of the resource(s) which is not expected to
+                be part of a workflow under test, and at the same time, the
+                timeout itself is large.
 
         Returns:
             bool: True in case all resources reached desired condition,
                 False otherwise
 
         """
+        if condition == error_condition:
+            # when this fails, this method is used in a wrong way
+            raise ValueError(
+                "Condition we are waiting for must be different from error"
```

Comment:
change "error" to "condition"

Reply:
fixed in e291865453ea60a6753d1fa2abfacbf2f4c699dc